src/routes/api/v1/follow.js

```diff
@@ -10,5 +10,14 @@ router
   .all(asyncHandler(checkUsername))
   .post(asyncHandler(FollowController.follow))
   .delete(asyncHandler(FollowController.unFollow));
+  
+router
+  .route('/:username/follower')
+  .get(asyncHandler(checkUserFromParams), asyncHandler(FollowController.getFollowers));
+
+router
+  .route('/:username/following')
```

Comment:
@verdotte `/followers` and `/followings`

Reply:
Noted